### ISSUE TITLE
Transphobic name and email policy

### ISSUE THREAD

**Issue by dippedrusk** [openreview/openreview#28]
**Describe the bug**
Since you can only add new names and emails and cannot delete old ones if they are still in use in some of your papers, and since these are also revealed externally (all names and email domain names publicly without login), trans people are visibly deadnamed. The name case is easy to see, the email case could happen if someone uses a custom domain name (e.g., me@firstnamelastname.com, as an example).

**To Reproduce**
1. Add a new name to an existing profile which has papers with your old name
2. Make the new name your preferred name
3. Try to delete the old name [it doesn't work]

**Expected behavior**
In accordance with these [principles for trans-inclusive publishing](https://publicationethics.org/news/vision-more-trans-inclusive-publishing-world), I should be able to just erase an old name and use a new one. I recognize that I have a publication on there that hasn't been updated yet, but the names you display don't *have* to be *visibly* linked to that so that I as a trans person have to synchronize all my name changes. Re emails, I understand that there are technical reasons for requiring confirmed emails (conflict detection / paper coreference, per your FAQ), however there should *at least* be an option to hide them entirely from public view.

**Screenshots**
![Screen Shot 2022-05-04 at 6 19 00 PM](https://user-images.githubusercontent.com/15020857/166725762-c0dcef18-4e28-44b6-bd48-b74aa70c7bd3.png)

**Device information (please complete the following information):**
 - OS: Mac
 - Browser: Firefox Nightly
 - Version: 101.0a1

**Additional context**
I would also recommend adding a section to your FAQs on trans name changes which are often distinct from people who change a name and don't mind keeping the old one around, specifically because of this issue of deadnaming and outing. Please reach out at me@dippedrusk.com if you have questions about trans inclusive policies / wording around this, I would be happy to help. This is an issue that affects a number of scholars, and it would be great to make the process easier for someone to do on their own / without emailing people, and more transparent.

**Comment by melisabok** [comment]
Currently the dead names are removed by one of the members of the support team. When names are associated to submissions we need to request permission to the Program Chairs of the venue to make that change. 

We currently have an entry in our FAQ: https://openreview.net/faq#question-remove-email if you think we should be more clear about the policy for trans name changes, please suggest the changes! Thanks for reporting this.

**Comment by vgautam** [comment]
There are issues with the current process and it should be changed. The first of the linked guiding principles of trans-inclusive publishing is accessibility, and it says, "Name changes should be available to authors upon request without legal documentation, unnecessary barriers, burdens, or labor placed upon the author making the request." The fourth is expedience and simplicity, and it recommends processes with a minimum of bureaucratic overhead.

I'd say needing to email the support team to remove (or even hide!) my own name in my own account on OpenReview constitutes an unnecessary barrier/burden/labour. My ideal proposed solution to this is to remove that barrier and let me delete a name by myself. My second choice solution is to add a button in the UI to let me hide a name from other people. The second choice applies to emails as well.

In addition, I have to point out that it's particularly bad to require a Program Chair (some random person, who could potentially be transphobic) to sign off on MY name change on MY account and MY publication as shown in OpenReview. This outs me to them, completely unnecessarily, and once again, is a bureaucratic barrier - what if they don't respond to their emails, for example? Or decide to weaponize their power? Their may be technical barriers to some of this but I think it it is a mistake to have a policy that requires someone else to sign off on what's mine.

As for the documentation, to me it sounded more about typos right now with "a name that does not belong to you" - so I would reword as follows:
> If your profile contains a confirmed email or a name that does not belong to you or no longer belongs to you (e.g., a deadname), please contact the OpenReview team by emailing [info@openreview.net](mailto:info@openreview.net).

However, as I've said, I really think you should reconsider the current policy which will probably require some technical changes but would make this process so much simpler and more trans-inclusive.

**Comment by melisabok** [comment]
Thank you for your valuable feedback. 

We understand that our current process of changing a name could be easier in OpenReview and we are open to improving it with feedback from the community.

We are working on a technical solution that allows users to change names without contacting us via email and we will be in touch to request feedback when this is ready. One thing we need to consider is the potential misuse of this feature where users change the name on their submission to fake or anonymized names after the review process has ended, for example to avoid having a rejected submissions be associated to them.

In the meantime we have added more details about the procedure to make changes in names and emails here: https://docs.openreview.net/getting-started/creating-an-openreview-profile/name-and-email-removal-procedure

**Comment by djsutherland** [comment]
I just want to weigh in a bit as someone who did get my deadname removed off of several OpenReview papers last year (a few ICLR papers + various account things – thanks again, Melisa) as well as a bunch of papers at other ML-relevant places (NeurIPS, PMLR which does ICML/AISTATS/..., ACM, IEEE, AAAI, arXiv):

- Every other place I talked to certainly required some human interaction from people at the publisher. An automated system would be amazing, but there are some bits (particularly editing PDFs) where it seems likely that someone is going to need to at least look at a diff.

  For the "attack model" of people disowning their rejections, I guess this mainly only matters if they only have rejections on the platform, because once they have any accepted papers they'd be disowning those too. Personally, I think having someone on OpenReview staff look over the list of name changes, especially if there are multiple requests with the same "source" name, seems reasonable. This can of course still be done if the name change is otherwise automated.

- Getting permission from program chairs is absolutely _not_ at all in line with what other venues require. No other venue where I changed my published papers required this. In my case, the relevant PCs did approve quickly and so I just more or less let it go, but consider:
    - What if a program chair isn't available (all have retired / passed away / ...), or just doesn't reply?
    - What if a program chair happens to just be very transphobic and says no? (This thankfully isn't the norm among the kinds of academics who are typically PCs of ICLR/etc, but there are certainly some around in very relevant fields; there was a recent big blow-up about pronouns in a collaborative PL/type theory book.)

   In either of these cases, does it really make sense for OpenReview to decide that it needs to enforce harm on a trans author, for...what purpose exactly?

   PCs run the process of deciding what belongs in a conference's proceedings in a way that is, hopefully, independent of the identity of the authors. If we were talking about changes to the scientific content of the papers, that would be one thing. But potentially not allowing changes based on a PC's wishes is analogous to saying that it'd be okay for the PC to decide to admit a paper to their conference if the author is [this cis-presenting person], but not _an identical paper_ if the author is [this trans-presenting person]. Is OpenReview really okay with that?

**Comment by melisabok** [comment]
Thanks @djsutherland for your feedback!

We had a discussion with the team and we decided the following workflow:

1. We will provide an interface in the profile edit page where the users can request the deletion of the name.
2. Someone of the team will review the request and make a decision of accepting or declining the request. 
3. When the request is accepted we will delete the name from the profile and update all the references to the deleted name with the preferred name listed in the profile. If the reference is a submitted paper through OpenReview, the team will contact the user and request the new PDF with name change. OpenReview will upload a new version of the PDF verifying the changes with a diff tool. 
All these changes will be made without asking permission to the Program Chairs. 

We will write a statement in the OpenReview site to notify the Program Chairs that OpenReview reserves the right of changing the submission author names if needed. 

Let us know what you think.

**Comment by djsutherland** [comment]
Personally, this policy seems to me pretty good and a notable improvement (as long as it’s handled promptly, which my previous request was so I have no particular worries there). Thanks for taking the feedback into account!

For reference, the main thing that some other publishers offer that this does not is a pathway to update citations in _others’_ papers; IEEE now does this without asking the other authors, ACM does as well but only with the other authors’ permission. They both do it by directly editing pdfs, which can be tricky.

**Comment by melisabok** [comment]
OpenReview doesn't do any process on the citations but we will take this into account. Maybe we need to start processing the PDF content.

**Comment by vgautam** [comment]
@melisabok I'm happy with this updated policy as well - it's a huge improvement from before and I really appreciate the prompt response to this feedback and your willingness to change the policy.

Re points 2 and 3, I'd add that it would be great if you could give an indication of the timeline for a team member to accept or decline a name change request.

**Comment by melisabok** [comment]
Our team is very responsive and we usually respond within the 24 hours during business days. We could indicate the timeline to the user when the name deletion is requested.

**Comment by melisabok** [comment]
Hi all, happy new year!

The profile page now provides a button to request the name deletion. Our support team moderate the request and accept and decline it. 

If the request is accepted, all the existing publications that contain the name to delete are replaced with the preferred name. 

I think the issue can be closed. Let us know what you think. Thanks!

**Comment by dippedrusk** [comment]
I'm thrilled to hear it - what a great new year's present! Thank you for listening and changing your policy. I will close this issue now.
